onnxruntime/core/providers/nnapi/nnapi_builtin/model.h

```diff
@@ -165,12 +165,15 @@ class Execution {
   };
 
  public:
-  explicit Execution(ANeuralNetworksExecution& execution, const Shaper& shaper);
+  explicit Execution(ANeuralNetworksExecution& execution /* , const Shaper& shaper */);
   ~Execution();
   Execution(const Execution&) = delete;
   Execution& operator=(const Execution&) = delete;
 
-  const Shaper& GetShaper() const { return shaper_; }
+  // TODO: Before we validate if we actually need to keep a shaper instance for Execution (if we have dynamic shape
+  // outputs, shape can get updated during execution),
+  // we commented out Shaper here for now.
+  /* const Shaper& GetShaper() const { return shaper_; } */
```

Comment:
Commented out the Shaper instance in Execution for now, due to the lack of support for updating dynamic dimensions in NNAPI (meaning that Execution temporarily is not required to keep a shaper instance for updating shapes.)